src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -98,20 +98,15 @@ const ReportActionItemFragment = (props) => {
                         )
                 );
             }
-            let {html, text} = props.fragment;
+            const {html, text} = props.fragment;
 
             // If the only difference between fragment.text and fragment.html is <br /> tags
-            // we replace them with line breaks and render it as text, not as html.
+            // we render it as text, not as html.
             // This is done to render emojis with line breaks between them as text.
```

Comment:
I'm guessing that rendering something with `<Text>` instead of `<RenderHTML>` is more performant too.